docs/install/installing-istio.md

```diff
@@ -34,7 +34,7 @@ cover a few useful Istio configurations and their benefits.
 
 ### Choosing an Istio installation
 
-You can install Istio with or without a service mesh:
+You can install Istio with or without sidecar proxies:
```

Comment:
```suggestion
You can install Istio with or without sidecar injection:
```

Reply:
Just to make this consistent with the next line.

Reply:
@fai555 will you fix this?

Reply:
@nak3 Fixed it.